pkg/chain/ethereum/tbtc.go

```diff
@@ -478,9 +492,12 @@ func (ta *tbtcApplication) FundingInfo(
 	if err != nil {
 		return nil, err
 	}
+
+	transactionHash, outputIndex := parseUtxoOutpoint(fundingInfo.UtxoOutpoint)
```

Comment:
Let's add an empty line.

```suggestion
	transactionHash, outputIndex := parseUtxoOutpoint(fundingInfo.UtxoOutpoint)
	
```